Samples/Excel-Get-Started-With-Dev-Kit/RUN_WITH_EXTENSION.md

```diff
@@ -0,0 +1,45 @@
+# Run the add-in from the Office Add-ins Development Kit
```

Comment:
Given that we consolidated the main readme, do we still need this file?

Reply:
Mingjia has proposed a way to deal with the duplicated content. Pls check.